Sources/UIExtensions/UIKitExtension.swift

```diff
@@ -2,6 +2,12 @@
 import UIKit
 
 public extension UITableView {
+    #if swift(>=4.2)
+    // Do nothing. UITableView has a RowAnimation enum
```

Comment:
I think this comment is not needed.
```swift
#if swift(>=4.2)
#else
typealias RowAnimation = UITableViewRowAnimation
#endif
```
After fixed, please squash your commits into one commit. (git rebase -i)

Reply:
Done.